doc/flame_fire_atlas/fire_atlas.md

```diff
@@ -0,0 +1,98 @@
+# FireAtlas
+
+FireAtlas is a tool for handling `texture atlases`. Using FireAtlas we can access images using keys
+from texture atlas like accessing key-value pair.
```

Comment:
```suggestion
FireAtlas is a tool for handling texture atlases. With FireAtlas one can access images and animations
stored in a `.fa` texture atlas by referring to them by their named keys.
```
"texture atlases" should not be in a code block.
The second sentence was not grammatically correct (check your sentences with grammarly or something similar if you are not certain they are correct).